app/assets/stylesheets/layouts/print_protocol.scss

```diff
@@ -225,3 +230,16 @@ hr {
 .ht_clone_top_left_corner {
   display: none;
 }
+
+.sa-type {
```

Comment:
is this class used anywhere?

Reply:
Yes. It is used for smart annotations style.